src/semantic_csv/impl/core.cljc

```diff
@@ -74,13 +75,14 @@
 (defn- subset-map [m ks]
   (into {} (map (fn [k] [k (get m k)]) ks)))
 
-(defmacro clone
```

Comment:
Ahh... Of course. Gotcha. If possible though, I'd like to support bootstrapped cljs, and it seems this would prevent that? Do you think there's a way to salvage clone in that setting, or should we just do a manual redef for cljs? Or is there a way that we can detect if we're in self-compile mode and decide whether to try running the macro or not?